D-lama-service/d-lama-service/Controllers/DataPointController.cs

```diff
@@ -336,13 +328,50 @@ public async Task<IActionResult> RemoveLabelDataPoint(int projectId, int dataPoi
             return Ok();
         }
 
+        /// <summary>
+        /// Gets all information of the labeled datapoints of a project.
+        /// </summary>
+        /// <param name="projectId"> The id of the project. </param>
+        /// <returns> Statuscode 200 with the list of information about the datapoints, if the project is found and the owner made the request, else statuscode 400. </returns>
+        [AdminAuthorize]
+        [TypeFilter(typeof(RESTExceptionFilter))]
+        [HttpGet("{projectId:int}/GetLabeledData")]
+        public async Task<IActionResult> GetLabeledDataForProject(int projectId) 
+        {
+            var project = await GetProjectWithOwnerCheckAsync(projectId, e => e.DataPoints);
+            var labeledDataModelList = new Dictionary<int,LabeledDataPointModel>();  
+            foreach (var dataPoint in project.DataPoints) 
+            {
+                var dataPointStats = await _unitOfWork.LabeledDataPointRepository.GetLabeledDataPointStatisticAsync(dataPoint.Id);
+                labeledDataModelList.Add(dataPoint.Id, new LabeledDataPointModel(dataPointStats));
```

Comment:
Wouldn't it be better to add the DataPointIndex as key instead of the Id? The use of the DataPointIndex would be more helpful for the client, as all the other endpoints take the DataPointIndex from the client.

![grafik](https://github.com/d-lama/d-lama-service/assets/79634084/03765308-c993-4604-b7b6-6af3145d48bc)

Reply:
Thanks Gianbello @guentgia !
Yes, I can do that... Now should we already merge this PR as I will need to adapt this method anyway to fit the suggestions of our customer?

Reply:
Ok, let us change that in a further commit! I approve this PR now